ui/src/dashboards/containers/DashboardPage.js

```diff
@@ -311,6 +284,28 @@ class DashboardPage extends Component {
     dashboardActions.putDashboardByID(dashboardID)
   }
 
+  handleSaveTemplateVariables = async templates => {
```

Comment:
could you confirm that this won't ever be invoked if a user has the Viewer role and does not have SuperAdmin status?